config/locales/en.yml

```diff
@@ -84,6 +84,8 @@ en:
     wrapup: "Next steps after deployment"
 
   page_subtitle:
+    hana_ha_enabled_info: |
+      Enable HA clustering and System Replication
```

Comment:
I was thinking that `clustering` word might redundant here.

` Enable HA and System Replication`

Reply:
It is not redundant. `HA clustering` stand for `High availability clustering`. `HA` by itself might not mean anything about clustering.